internal/api/container/remove.go

```diff
@@ -0,0 +1,113 @@
+package container
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/projecteru2/barrel/internal/sock"
+	"github.com/projecteru2/barrel/internal/utils"
+	minions "github.com/projecteru2/minions/lib"
+	log "github.com/sirupsen/logrus"
+)
+
+var regexDeleteContainer *regexp.Regexp = regexp.MustCompile("/(.*?)/containers/(\\w+)(\\?.*)?")
```

Comment:
```regexp.MustCompile(`/(.*?)/containers/(\w+)`)```
no escape char may be readable, and the third group doesn't refer, so the last ```(\?.*)?``` could be deleted?

Reply:
the raw string style will adopt.
yes the last could be deleted, i wrote (\?.*)? 'cause in js and java both use a full string matching other than partial matching, I wrote a case some minites before and found that FindStringSubmatch is partial matching.

Reply:
the raw string style will adopt.
the last part is used to do a full url string matching so I think it's better to keep it. so as not to match other pattern like `/v1.40/containers/some-id-here/some-subres-here/action-here`, althrough currently there are no such pattern in docker apis.

Reply:
the raw string style will adopt.
yes, it could be deleted, i think i should do a full url string matching but \w will not match a '/', and MatchAllSubString will partial match string, not liking in java and js will do a full string matching.

Reply:
the raw string style will adopt.
well i think i should use `^/(.*?)/containers/(\w+)(\?.*)?$` , will fix it

Reply:
正则已经更正了